src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx

```diff
@@ -1,19 +1,40 @@
+import React from "react";
 import styled from "styled-components";
 
+import { deviceBreakpoints } from "@src/design-system/theme/lightTheme";
+
 interface ConfirmationBoxProps {
   open: boolean;
   preventAnimation: boolean;
+  children: JSX.Element;
 }
 
-export const ConfirmationBox = styled.div<ConfirmationBoxProps>`
+export const ConfirmationBox: React.FC<ConfirmationBoxProps> = ({
+  open,
+  preventAnimation,
+  children,
+}) => {
+  return (
+    <Box open={open} preventAnimation={preventAnimation}>
+      {children}
+    </Box>
+  );
+};
+
+const Box = styled.div<ConfirmationBoxProps>`
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 4rem 2rem 3rem;
   position: fixed;
-  left: 0;
+  left: 41%;
   background-color: ${({ theme }) => theme.colors.background};
-  width: 100%;
+  width: 52.8rem;
```

Comment:
Could you try `width: auto` instead?

Reply:
The problem with `width: auto;` is that the box's size will vary according to the content, so we will have different sizes for `<PrimaryConfirmationBox />` and `<DeleteConfirmationBox />`

Reply:
Well, is that an issue?

Reply:
Not really actually, I was just following the wireframe. A fixed `width` would help me align the Box as much as possible and whatever the content though.

With `width: auto;` it seems that the `<DeleteConfirmationBox />` is out of line (even though without the Bow wasn't perfectly aligned).

![image](https://user-images.githubusercontent.com/33835824/97205352-a931cf80-17b7-11eb-856d-7547b1ca6220.png)

What is your opinion?

Reply:
Ok,we'll refacto everything later then.